exampleSite/config.toml

```diff
@@ -1,6 +1,6 @@
 # Site settings
 baseurl = "/"
-languageCode = "en-us"
+LanguageCode = "en-us"
```

Comment:
It can be lowercase here, but needs to be uppercase in the html layouts afaik.

Reply:
Fixed in 02991898802d2faa7c8be867619a655643a2edb9